src/rust/multi.comick/src/parser.rs

```diff
@@ -148,7 +148,7 @@ pub fn parse_manga_list(
 		}
 		Ok(MangaPageResult {
 			manga: mangas,
-			has_more: false,
+			has_more: true,
```

Comment:
this shouldn't always be true, otherwise it'll loop infinitely. you need to have a check (e.g. if `mangas` is empty) to see if there are more or not.

Reply:
fiixed please check now